source/source.go

```diff
@@ -233,12 +235,23 @@ func endpointsForHostname(hostname string, targets endpoint.Targets, ttl endpoin
 	var endpoints []*endpoint.Endpoint
 
 	var aTargets endpoint.Targets
+	var aaaTargets endpoint.Targets
```

Comment:
Is this intentionally `aaa` and not `aaaa` (with the extra `a`)?

Reply:
I can change it to be properly named, but there's no absolute reason why that's not quad a.

Technically doesn't matter, but can make it harder to read so should that be corrected to quad a (aaaa)?